atd-vze/src/views/Users/Users.js

```diff
@@ -96,7 +96,7 @@ const UserRow = ({ user }) => {
           ? format(parseISO(user.last_login), "MM/dd/yyyy")
           : "Never"}
       </td>
-      <td>{rules[user.app_metadata.roles[0]].label}</td>
+      <td>{rules[user.app_metadata.roles[0]]?.label}</td>
       <td>
```

Comment:
@frankhereford one really minor change. how about adding a fallback text string here so it doesn't render `undefined`. i think rendering `"Error"` would suffice. Just to make it clear that something went horribly wrong 🤷